docs/guides/singlestore/quickstart/quickstart.md

```diff
@@ -0,0 +1,519 @@
+---
+title: SingleStore Quickstart
+menu:
+  docs_{{ .version }}:
+    identifier: sdb-quickstart-quickstart
+    name: Overview
+    parent: sdb-quickstart-singlestore
+    weight: 15
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# SingleStore QuickStart
+
+This tutorial will show you how to use KubeDB to run a SingleStore database.
+
+<p align="center">
+  <img alt="lifecycle"  src="/docs/guides/singlestore/quickstart/images/singlestore-lifecycle.png">
+</p>
+
+> Note: The yaml files used in this tutorial are stored in [docs/guides/singlestore/quickstart/yamls](https://github.com/kubedb/docs/tree/{{< param "info.version" >}}/docs/guides/singlestore/quickstart/yamls) folder in GitHub repository [kubedb/docs](https://github.com/kubedb/docs).
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Now, install KubeDB cli on your workstation and KubeDB operator in your cluster following the steps [here](/docs/setup/README.md).
+
+- [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) is required to run KubeDB. Check the available StorageClass in cluster.
+
+  ```bash
+  $ kubectl get storageclasses
+  NAME                 PROVISIONER             RECLAIMPOLICY     VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+  standard (default)   rancher.io/local-path   Delete            WaitForFirstConsumer   false                  6h22m
+  ```
+
+- To keep things isolated, this tutorial uses a separate namespace called `demo` throughout this tutorial.
+
+  ```bash
+  $ kubectl create ns demo
+  namespace/demo created
+  ```
+
+## Find Available SingleStoreVersion
+
+When you have installed KubeDB, it has created `SinglestoreVersion` crd for all supported SingleStore versions. Check it by using the following command,
+
+```bash
+$ kubectl get singlestoreversions
+NAME     VERSION   DB_IMAGE                                                          DEPRECATED   AGE
+8.1.32   8.1.32    ghcr.io/appscode-images/singlestore-node:alma-8.1.32-e3d3cde6da                72m
+8.5.7    8.5.7     ghcr.io/appscode-images/singlestore-node:alma-8.5.7-bf633c1a54                 72m
+
+```
+## Create SingleStore License Secret
+
+We need SingleStore License to create SingleStore Database. So, we will pass the license by a secret.
```

Comment:
So, Ensure that you have acquired a license and then simply pass the license by secret.